### ISSUE TITLE
[Bug Report] V-Dialog Regression BreaksTinyMCE Inside Dialog

### ISSUE THREAD

**Issue by DanielStout5** [vuetifyjs/vuetify#7608]
### Versions and Environment
**Vuetify:** 1.5.8
**Last working version:** 1.5.7
**Vue:** 2.6.10
**Browsers:** Firefox 65.0
**OS:** Windows 10

### Steps to reproduce
Put TinyMCE inside a v-dialog

Open any dialog from TinyMCE (such as to edit a link) and you won't be able to focus on any of the inputs (e.g. to edit the URL).

It used to work in Vuetify 1.5.7, and has been broken since 1.5.8.

I think this commit may be the cause of the issue: https://github.com/vuetifyjs/vuetify/commit/dbe3213 but that's just a guess

### Expected Behavior
You should be able to focus on dialogs opened by a TinyMCE editor inside a dialog (Just like how it worked for all versions before 1.5.8)

### Actual Behavior
The input fields cannot be focused on

### Reproduction Link
https://codepen.io/dstout-immediac/pen/vqxXRR

**Comment by johnleider** [comment]
You've submitted an issue that does not contain a reproduction. In order to effectively disposition this, we need to verify that the problem exists **outside** of your environment. This issue will be tagged as `needs reproduction` for the next 14 days before being closed.

You can find information on how to contribute to vuetify [here](https://github.com/vuetifyjs/vuetify/blob/master/.github/CONTRIBUTING.md).

Services to create a reproduction:

- [Codepen](https://template.vuetifyjs.com)
- [Codepen w/Router](https://codepen.io/zikeji/pen/ypeQNm)
- [Codesandbox](https://codesandbox.io)
- [JSFiddle](https://jsfiddle.net/)
- A small reproduction repository

Thank you for your contribution and interest in improving Vuetify.

**Comment by DanielStout5** [comment]
I've added a reproduction link:

https://codepen.io/dstout-immediac/pen/vqxXRR


Just click the Show button, then Insert -> Link, and see that you can't focus on the Url, Text, or Title fields. This worked fine in Vuetify 1.5.7 and below.

**Comment by iainsharris** [comment]
When I try your reproduction I am able to type into the fields.

**Comment by DanielStout5** [comment]
What browser/OS are you using? Because it doesn't work for me on two different Windows machines (tested in Chrome and Firefox) or Chrome android, and both a client and a coworker experienced the same issue after our software was published with this bug.

Are you sure you're testing the right thing? Here's a gif demonstrating the issue:

![](https://i.imgur.com/vge4fRP.gif)

I should be able to type into those fields, but it doesn't allow focus

**Comment by jlucloran** [comment]
Hello.
I am glad someone else has this issue.
I also have it when using CKEditor5 with their vue.js integration.
Everything works fine, but when I insert an editor in a v-dialog, I can't focus on the text fields to edit a link. It automatically focuses on the 1st element in the v-dialog (in this case another v-text-field).
I am on vuetify 1.5.5.

![image](https://user-images.githubusercontent.com/46494798/60214820-4384aa80-9834-11e9-8095-bd9e6bedafa2.png)

**Comment by iainsharris** [comment]
Strangely, when I tried your pen again today I got the same behaviour that you are reporting whereas yesterday I could definitely type into the fields in the 'Insert link' dialog.


I used Chrome on both occasions. I just tested with Edge and get the same problem.

On 26 June 2019 at 10:25 PM, Daniel Stout <notifications@github.com> wrote:



What browser/OS are you using? Because it doesn't work for me on two different Windows machines (tested in Chrome and Firefox) or Chrome android, and both a client and a coworker experienced the same issue after our software was published with this bug.

Are you sure you're testing the right thing? Here's a gif demonstrating the issue:

I should be able to type into those fields, but it doesn't allow focus
